base/random.jl

```diff
@@ -1596,6 +1618,30 @@ end
 # each element of A is included in S with independent probability p.
 # (Note that this is different from the problem of finding a random
 #  size-m subset of A where m is fixed!)
+
+"""
+    randsubseq!([rng::AbstractRNG=GLOBAL_RNG,] S, A, p)
+
+Like [`randsubseq`](@ref), but the results are stored in `S`
+(which is resized as needed).
+
+# Examples
+```jldoctest
+julia> b = zeros(4);
+
+julia> a = collect(1:8);
+
+julia> rng = MersenneTwister(1234);
+
+julia> randsubseq!(rng, b, a, 0.5);
```

Comment:
`rng` and `a` are used only once, I suggest to make it less verbose with:
`randsubseq!(MersenneTwister(0), b, 1:8, 0.5)` or even `randsubseq!(MersenneTwister(0), zeros(4), 1:8, 0.5)`.  (I find the `1234` distracting, I personally prefer `0`).